src/MusicalScore/Graphical/VexFlow/VexFlowMusicSheetDrawer.ts

```diff
@@ -53,15 +54,15 @@ export class VexFlowMusicSheetDrawer extends MusicSheetDrawer {
     public drawSheet(graphicalMusicSheet: GraphicalMusicSheet): void {
         // vexflow 3.x: change default font
         if (this.rules.DefaultVexFlowNoteFont === "gonville") {
-            (Vex.Flow as any).DEFAULT_FONT_STACK = [(Vex.Flow as any).Fonts?.Gonville, (Vex.Flow as any).Fonts?.Bravura, (Vex.Flow as any).Fonts?.Custom];
+            // rvilarl Vex.Flow.setMusicFont("Gonville","Bravura","Custom");
```

Comment:
Why is this commented out? Is it not working?

Reply:
Done I just forgot to uncomment.